eng/pipeline/variables/common.yml

```diff
@@ -13,11 +13,11 @@ variables:
   value: build-info/docker
 - name: publicGitRepoUri
   value: https://github.com/microsoft/go
-- name: officialRepoPrefix
+- name: officialRepoPrefixes
   ${{ if eq(parameters.nightly, false) }}:
     # Publish to "public/", which will be detected by MCR trigger.
-    value: public/
-  ${{ if eq(parameters.nightly, true) }}:
+    value: public/,private/internal/
```

Comment:
What is this change doing, and do we need to update the associated comment?

Reply:
Good call, I'll update the comment.

This is copied from a .NET Docker change to let MAR/MCR publish work for an internal-only destination. (We don't do this kind of publish, but it seems worth taking the change in case we do it in the future, and it doesn't hurt in the meantime.)

* https://github.com/dotnet/docker-tools/pull/1048